src/utils/staking.js

```diff
@@ -301,7 +301,7 @@ export class Staking {
                         contract
                     }
                     const fee = validator.fee = await validator.contract.get_reward_fee_fraction()
-                    fee.percentage = fee.numerator / fee.denominator * 100
+                    fee.percentage = +(fee.numerator / fee.denominator * 100).toFixed(2)
```

Comment:
why is `+` needed?

Reply:
@vgrichina the `+` converts it back to `number` since `toFixed` converts the result into `string`